.github/actions/setup-javascript/action.yml

```diff
@@ -12,4 +10,5 @@ runs:
         cache: 'yarn'
 
     - name: Install yarn dependencies
+      shell: bash
```

Comment:
[question] Do you know when specifying bash is necessary?

Reply:
You'd have to ask github. For whatever reason `run` steps default to bash when used in a workflow, but it's required to specify the shell when used in a `composite run steps` shared action.

Reply:
:sigh: just realized you said `when` and not `why`. The when is: anytime you're creating a shared action and anytime you want to use something other than bash.

Reply:
Ha, you actually answered my question with your first go! The docs I looked at were basically like "it uses bash by default except on Windows" and so I didn't really know why you were specifying it.  But, the composite run steps seems to be the answer!